assets/terraform-modules/azure/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -48,6 +48,9 @@ resource "null_resource" "copy-controller-secrets" {
 
   provisioner "remote-exec" {
     inline = [
+      "sudo mv $HOME/kubeconfig /etc/kubernetes/kubeconfig",
```

Comment:
We do not test azure, I think we can skip this changes, as we have no way of testing them anyway. When we try to bring Azure platform back, it will require a lot of many changes anyway.